docs/tutorials/Fundamentals.md

```diff
@@ -39,11 +39,11 @@ It is important to aknowledge the following facts regarding original raster data
 ## Protocols and data formats
 
 
-ITowns comes with a wide range of compatible data sources. 
-It can be used to display data from web servers that use protocols such as Web Map (Tile) Service ([WMS](https://www.ogc.org/standards/wms) and [WMTS](https://www.ogc.org/standards/wmts)), Web Feature Service ([WFS](https://www.ogc.org/standards/wfs)) and Tile Map Service ([TMS](https://wiki.osgeo.org/wiki/Tile_Map_Service_Specification)).
+ITowns comes with a wide range of compatible data sources (all inheriting from the `{@link Source}` base class).
```

Comment:
My point was to indicate to readers where to look to find the supported data formats and available data sources. Otherwise, there is no mention of `Source` or any of the specific sources in this section which is not good in my opinion (we should tell users where to look in the doc to find supported data sources).
I reformulated and added more details about the sources below, let me know if it's better?

Reply:
Indeed you're right on this point : it is interesting to specify where the usage of `Source` is documented.
The reformulation is fine for me :+1: